### DISCUSSION TITLE
Use recording on one camera to trigger recording on another

### DISCUSSION THREAD

**Discussion by grainsoflight** [blakeblackshear/frigate#9296] [Ask A Question]
Is there a method to trigger recording on one camera only when there is an event on another? My reolink trackmix is a two lens camera, wide and tele. Theres really no reason to run a detection process on the tele, as the wide already has one running. The wide cam is set to record 24/7, but in the spirit of saving resources, it would be great if the tele would only record when the wide senses an event. That way I dont have redundant recordings.

**Comment by NickM-27** [comment]
This can be done using custom events in 0.13

**Reply by grainsoflight** [reply]
Cool, its not super pressing to have at the moment so Ill wait for the release version. Thanks

**Reply by grainsoflight** [reply]
Could you by chance point me in the right direction as to where I might find details for this in the docs?

**Reply by grainsoflight** [reply]
Wait, I think I figured this out, If im correct the this config on the zoom cam:

record:
      enabled: True
      events:
        required_zones: 
          - zone_0
 
Should record whenever an event occurs in zone_0 even if it is out of field of view so long as the other camera also has a zone named "zone_0". That sound correct?

**Reply by NickM-27** [reply]
No, that is not correct. Like I said, need to use the custom event api https://docs.frigate.video/integrations/api#post-apieventscamera_namelabelcreate
